basis/ui/gadgets/theme/switching/switching.factor

```diff
@@ -0,0 +1,26 @@
+! Copyright (C) 2009, 2010 Slava Pestov, Joe Groff.
+! See http://factorcode.org/license.txt for BSD license.
+USING: fry hashtables help.stylesheet io.styles kernel
+namespaces ui.gadgets.theme ;
+IN: ui.gadgets.theme.switching
+
+: (update-style) ( style color elt -- )
+    '[ _ _ rot ?set-at ] change-global ;
+
+: update-stylesheet ( -- )
```

Comment:
Shouldn't the hooks take away the need to call ``update-stylesheet`` and ``(update-style)`` at all?